vo_models/uws/models.py

```diff
@@ -180,45 +178,45 @@ class JobSummary(BaseXmlModel, Generic[ParametersType], tag="job", ns="uws", nsm
     Parameters:
         job_id (JobIdentifier, str):
             (element) - The identifier for the job.
-        run_id (str):
+        run_id (str | None):
             (element) - This is a client supplied identifier - the UWS system does nothing other than to return it
             as part of the description of the job
-        owner_id (str):
+        owner_id (str | None):
             (element) - The owner (creator) of the job - this should be expressed as a string that can be
             parsed in accordance with IVOA security standards.
 
                         If there was no authenticated job creator then this should be set to NULL.
         phase (ExecutionPhase):
             (element) - The execution phase.
-        quote (UTCTimestamp):
+        quote (UTCTimestamp | None):
             (element) - A Quote predicts when the job is likely to complete.
-        creation_time (UTCTimestamp):
+        creation_time (UTCTimestamp | None):
             (element) - The instant at which the job was created.
 
                         Note that the version 1.1 of the specification requires that this element
                         be present. It is optional only in versions 1.x of the schema for backwards compatibility.
                         2.0+ versions of the schema will make this formally mandatory in an XML sense.
-        start_time (UTCTimestamp):
+        start_time (UTCTimestamp | None):
             (element) - The instant at which the job started execution.
-        end_time (UTCTimestamp):
+        end_time (UTCTimestamp | None):
             (element) - The instant at which the job finished execution.
-        execution_duration (int):
+        execution_duration (int | None):
             (element) - The duration (in seconds) for which the job should be allowed to run.
 
                         A value of 0 is intended to mean unlimited.
-        destruction (UTCTimestamp):
+        destruction (UTCTimestamp | None):
             (element) - The time at which the whole job + records + results will be destroyed.
-        parameters (Parameters):
+        parameters (Parameters | None):
             (element) - The parameters to the job (where appropriate)
-        results (Results):
+        results (Results | None):
             (element) - The results for the job
-        error_summary (ErrorSummary):
+        error_summary (ErrorSummary | None):
             (element) - A short summary of an error
-        job_info (list[str]):
+        job_info (list[str] | None):
             (element) - This is arbitrary information that can be added to the job description by the UWS
             implementation.
 
-        version: (UWSVersion)
```

Comment:
Did you figure out why `version` was showing `| None` in the sphinx docs even without the info in the docstring?

https://vo-models.readthedocs.io/latest/pages/api/uws_api.html#vo_models.uws.models.JobSummary

Reply:
Fixed by using the autodoc type hints.